developer-docs-site/docs/nodes/ait/connect-to-testnet.md

```diff
@@ -153,7 +153,8 @@ At this point you already used your owner account to initialized a validator sta
     ```
     aptos node join-validator-set \
       --pool-address <owner-address> \
-      --profile ait3-operator
+      --profile ait3-operator \ 
+      --max-gas 10000 # You can play with the amt of max gas here. Make sure you have sent your operator enough tokens for them to pay gas fee
```

Comment:
`You can adjust the amount of max gas here. Please make sure you have sent your operator enough tokens to pay for the gas fee.`